server/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -7,6 +7,8 @@
  */
 package org.elasticsearch.action.support;
 
+import joptsimple.internal.Strings;
```

Comment:
This seems a little random - I see below that you're using the join function. Maybe check out org.elasticsearch.common.Strings#collectionToCommaDelimitedString ?

Reply:
@dakrone you are right, usually IDE picks the right one so I didn't even check this time. Good catch!